<launch>
    <arg name="src_image" default="/sensing/camera/traffic_light/image_raw/compressed"/>
    <arg name="scale_num" default="1"/>
    <arg name="scale_denom" default="2"/>
    <arg name="imshow" default="false"/>
    <arg name="crop_range" default="[2240,1360,640,480]"/>
    <arg name="bayer" default="false"/>

    <node name="turbo_decode_node" pkg="turbo_decoder" exec="decoder_node" output="screen">
        <remap from="src_image" to="$(var src_image)"/>
        <remap from="proc_time" to="/turbo/proc_time"/>

        <param name="scale_num" value="$(var scale_num)"/>
        <param name="scale_denom" value="$(var scale_denom)"/>
        <param name="use_imshow" value="$(var imshow)"/>
        <param name="crop_range" value="$(var crop_range)"/>
        <param name="is_bayer" value="$(var bayer)"/>

        <param name="use_imdecode" value="false"/>
    </node>

    <node name="opencv_decode_node" pkg="turbo_decoder" exec="decoder_node" output="screen">
        <remap from="src_image" to="$(var src_image)"/>
        <remap from="proc_time" to="/imdecode/proc_time"/>

        <param name="scale_num" value="$(var scale_num)"/>
        <param name="scale_denom" value="$(var scale_denom)"/>
        <param name="use_imshow" value="$(var imshow)"/>
        <param name="crop_range" value="$(var crop_range)"/>
        <param name="is_bayer" value="$(var bayer)"/>

        <param name="use_imdecode" value="true"/>
    </node>
</launch>
